### ISSUE TITLE
If info in app.config is missing, GPX build will fail, make it optional

### ISSUE THREAD

**Issue by TimMcCauley** [GIScience/openrouteservice#80]

**Comment by TimMcCauley** [comment]
Let's make this optional ;-)

**Comment by rabidllama** [comment]
I think the problem is that if we are sticking to the gpx schema, than all of the extra info that is in the app.config is mandatory, so strictly speaking, if that information is not there, then what we return isn't gpx

**Comment by TimMcCauley** [comment]
Roger, shall we hard core something default which can be overwritten by app.config?

**Comment by MichaelsJP** [comment]
I added some logic inside the GpxRoutingResponseWriter to check if the config variables are set. if not they are left blank and an error is printed out to the user console e.g. "Config parameter support_mail is missing" . 

I omitted a classic try catch because the error handling breaks the gpx routine.
@TimMcCauley We could insert some generic values and/or skip the printing...

Generally speaking, all the missing values wouldn't break the gpx shema as long as the header is set correctly
